Packs/JWT/Integrations/JWT/JWT.py

```diff
@@ -0,0 +1,188 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from typing import Dict
+import jwt
+import uuid
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+
+    def request_access_token(self, headers, body):
+        response = self._http_request('post', headers=headers, json_data=body)
+        return response
+
+
+''' HELPER FUNCTIONS '''
+
+
+def encode_authentication_token(secret_key, jti=None, iss=None, aud=None, sub=None, scp=None, iat=None, exp=None,
+                                nbf=None, token_timeout=None, additional_claims=None):
+    token_id = str(uuid.uuid4())
+    jti = jti or token_id
+
+    token_timeout = token_timeout or '300'
+    now = datetime.utcnow()
+    timeout_datetime = now + timedelta(seconds=int(token_timeout))
+    epoch_time = int((now - datetime(1970, 1, 1)).total_seconds())
+    epoch_timeout = int((timeout_datetime - datetime(1970, 1, 1)).total_seconds())
+    exp = exp or epoch_timeout
+    iat = iat or epoch_time
+
+    claims = {
+        'exp': exp,
+        'iat': iat,
+        'jti': jti
+    }
+
+    if iss:
+        claims['iss'] = iss
+    if aud:
+        claims['aud'] = aud
+    if sub:
+        claims['sub'] = sub
+    if scp:
+        claims['iss'] = scp
+    if scp:
+        claims['nbf'] = nbf
+    if additional_claims:
+        claims.update(json.loads(additional_claims))
+
+    payload = jwt.encode(claims, secret_key, algorithm='HS256')
```

Comment:
```suggestion
    payload = jwt.encode(claims, secret_key, algorithm='HS256')
```
In my opinion its better to pass the algorithm as command arg instead of using hardcoded value of `HS256`
like `ES256 ` see the [documentation](https://pyjwt.readthedocs.io/en/latest/api.html)
but this is not a must

Reply:
nice!
please update the README file too with the new params and arguments you added(Scope,Issued At, Algorithm...)